### PR TITLE
Add different screens and reformat data for routing

### PR DESCRIPTION
**Pull request by laytonmaes**
<h1>PULL REQUEST</h1>

<ul>
  <li>What does it change?</li>
Adds routing and changes location of data to facilitate a multi page application.
</ul>

<h2>Type of Change</h2>

- [x] **New Feature**

<h2>How has this been tested?</h2>

- [x] **JS**
- [x] **Dev Tools**
- [x] **Website**

<h2>Checklist</h2>

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new bugs
- [x] I have tagged project partners for review & it has been reviewed 

<h2>Does this PR close any issues?</h2>

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,16 +1,102 @@
 import { StatusBar } from 'expo-status-bar';
-import React from 'react';
-import { StyleSheet, View } from 'react-native';
+import React, {useState} from 'react';
```
Comment (BlueNealis) [inline App.js:2]: I see so has app.js become kinda like the functional blood/neurosystem of the application ?  Where the views are like a index of the components filled with that app blood.

**Review by BlueNealis — APPROVED** [review]
I think it makes sense how you moved things around! Thank you for setting up navigation.

**Review by Neil-B-Linton — APPROVED** [review]
Awesome job with doing the routing!

**Review by ameliaeiger — APPROVED** [review]
Awesome, will need to change to a modular file structure by removing individual components to their own files that we will export but shouldn't be a tough fix.

**Pull request merged**